.github/workflows/build_and_test.yml

```diff
@@ -233,15 +236,22 @@ jobs:
     runs-on: ${{ matrix.os-version }}
     strategy:
       matrix:
-        os-version: [macos-12, macos-13]
-        arch: [x86_64, arm64]
-        # Handle ZSTD build by Cirrus CI
-        # config-zstd: [ON, OFF]
-        config-zstd: [OFF]
-        exclude:
-          # excludes ZSTD on Arm64
-          - arch: arm64
+        include:
+          - os-version: macos-12
```

Comment:
no macos-14? maybe we should just maintain 13 and 14?

Reply:
macOS 14 is M1 only: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

macOS 12 and 13 are also x86_64

Reply:
@seladb  I see, then don't we test zstd on macos 14 arm?

Reply:
I don't think it's necessary to test Zstd on all archs. 

I tried on arm64 and it [didn't work](https://github.com/seladb/PcapPlusPlus/actions/runs/9631691995/job/26563813940), probably because the Zstd version installed on this machine isn't compiled with arm64, so x86_64 is currently the only option